libraries/Microsoft.Bot.Connector/Authentication/IOAuthConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.Connector.Authentication
+{
+    /// <summary>
+    /// Configuration for OAuth client credential authentication.
+    /// </summary>
+    public interface IOAuthConfiguration
+    {
+        /// <summary>
+        /// OAuth Authority for authentication.
+        /// </summary>
+        string Authority { get; set; }
+
+        /// <summary>
+        /// OAuth scope for authentication.
+        /// </summary>
+        string Scope { get; set; }
+    }
+
+    public class OAuthConfiguration : IOAuthConfiguration
```

Comment:
Silly but... a unit test here would be good.

Reply:
I removed the interface and now its a property only class, I don't think it needs tests. Feel free to argument for the opposite case :)